extensions/vault/runtime/src/main/java/io/quarkus/vault/runtime/client/dto/totp/VaultTOTPCreateKeyBody.java

```diff
@@ -0,0 +1,47 @@
+package io.quarkus.vault.runtime.client.dto.totp;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import io.quarkus.vault.runtime.client.dto.VaultModel;
+
+public class VaultTOTPCreateKeyBody implements VaultModel {
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public Boolean generate;
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public Boolean exported;
+
+    @JsonProperty("key_size")
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public Integer keySize;
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public String url;
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public String key;
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public String issuer;
+
+    @JsonProperty("account_name")
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public String accountName;
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public String period;
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public String algorithm;
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public Integer digits;
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public Integer skew;
+
+    @JsonProperty("qr_size")
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public Integer qrSize;
+
+    public boolean isProducingOutput() {
+        // When exported is not set, by default is true
+        return (exported == null || exported == Boolean.TRUE) && (generate != null && generate == Boolean.TRUE);
```

Comment:
`(exported == null || exported) && (generate != null && generate)`
it is simpler, and the logic is not broken when the user passes `new Boolean(true)` instead of `Boolean.TRUE`
or (a bit longer but a bit clearer):
```
    public boolean isProducingOutput() {
        return is(exported, true) && is(generate, false);
    }

    private boolean is(Boolean v, boolean defaultValue) {
        return v == null ? defaultValue : v;
    }
```

Reply:
Good catch thanks.